website/docs/enterprise/getting-started/install-enterprise-cli.mdx

```diff
@@ -85,19 +87,26 @@ Please use the following command to start the installation wizard of Weave GitOp
 `gitops-ee bootstrap` is a workflow that will take you through the following stages:
 
 1. [Verify Flux](#verifying-flux): verify Flux installation on the Management cluster.
-2. [Verify Entitlement](#verifying-entitlement): verify the Entitlements secret content (username, password, entitlement).
-3. [Configure Git Access](#configure-git-access): configure the access to your configuration repo.
-4. [Select WGE version](#selecting-wge-version): from the latest 3 available releases.
-5. [Create Cluster User](#create-cluster-user): create a Secret with the username and password for the emergency cluster user.
-6. [Configure Dashboard Access](#configure-dashboard-access): choose between 2 methods to access the dashboard either local or external.
-7. [Access the dashboard](#access-the-dashboard): via the link from the installation success message.
-8. (Optional) [Configure OIDC](#configure-oidc): to enable login to dashboard via OIDC providers.
+2. (Optional) [Bootstrap Flux](#bootstrap-flux): bootstrap flux in case flux not found.
+3. [Verify Entitlement](#verifying-entitlement): verify the Entitlements secret content (username, password, entitlement).
```

Comment:
it feels odd that we could be doing a mutation in the cluster like installing flux without first verifying entitlements (given that is a pre-requirement). how do you guys feel about this ( cc @MostafaMegahid @waleedhammam )

Reply:
i think when we discussed it before, there was a dependency as the entitlement lies in `flux-system` namespace

Reply:
Yes, I mean that given we have verifyXX as step, those seem better to come first before any action, as they are pre-conditions that we need to meet or we would drop. if we are in the scenario of dropping cause pre-condition is not met, you dont want to have done any change in the users environment
- verifies first then 
- flux

<img width="761" alt="Screenshot 2023-11-21 at 08 18 21" src="https://github.com/weaveworks/weave-gitops/assets/12957664/124720c1-663c-4d5c-b6b4-8e29f00d15ed">